pkgs/by-name/re/resnap/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  pkgs,
+  fetchFromGitHub,
+  writeShellApplication,
```

Don't use `pkgs` inside the derivation. Instead, add the dependencies to the inputs.

```suggestion
  lib,
  fetchFromGitHub,
  writeShellApplication,
  ffmpeg,
  feh,
  imagemagick_light,
  lz4,
```